controllers/article-controller.js

```diff
@@ -0,0 +1,14 @@
+const { fetchArticleById } = require("../models/article-model");
+
+function getArticleById(req, res, next) {
+  const { article_id } = req.params;
+  fetchArticleById(article_id)
+    .then((response) => {
+      res.status(200).send(response);
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({article: response})